test/jasmine/tests/image_test.js

```diff
@@ -444,7 +500,7 @@ describe('image hover:', function() {
                 zmax: [1, 1, 1],
                 text: [['A', 'B', 'C'], ['D', 'E', 'F']],
                 hovertemplate: '%{text}<extra></extra>'
-            }], layout: {width: 400, height: 400}};
+            }], layout: {width: 400, height: 400, yaxis: {scaleanchor: null}}};
```

Comment:
Setting `scaleanchor: 'null'` is fine here (adapting the pixel coordinates to hover on is annoying :smirk: ), but can you add a comment above this explain why `scaleanchor` is set to `null`? Thanks!

Reply:
I actually went ahead and removed the `scaleanchor`. Guessing the new hover position wasn't so hard :smile_cat: 

See d71c3b345d2eecb7b3087c383f00f5f102b9c2f6

Reply:
Even better. Thanks!!